Include/internal/pycore_frame.h

```diff
@@ -76,6 +76,9 @@ typedef struct _PyInterpreterFrame {
     uint16_t return_offset;  /* Only relevant during a function call */
     char owner;
     char visited;
+#ifdef Py_DEBUG
```

Comment:
This will make the frame a different size in debug builds. We try to avoid that as it can cause issues when extensions built for debugging are loaded by a release Python or vice-versa.

I'd suggest this:
```C
#ifdef Py_DEBUG
     uint8_t visited:7;
     uint8_t lltrace:1;
#else
     uint8_t visited;
#endif
```

Reply:
lltrace is a range of values from 0 to 5. So we need to split :4 instead.